### DISCUSSION TITLE
帮助龙猫做联机

### DISCUSSION THREAD

**Discussion by Enigma-Soul** [Meloong-Git/PCL#1615] [讨论]
这个早就想搞了
但是一直没时间
不需要打洞
主要原理是用IP6设备都有一个公网IP
源码我用的PYTHON
IP6端口：
[IP6公网地址]:端口

**Comment by Chiloven945** [comment]
![image](https://user-images.githubusercontent.com/116699482/230720389-8ffab849-b6af-4b84-abd1-dc2e2efc8708.png)
有些人可能会没有ipv6地址，或者他的ipv6地址是内网的，我连我家网就没有公网的ipv6他的地址是fe开头的，这图还是我开热点出来的
现在用ipv6还是太冒险了

**Reply by Enigma-Soul** [reply]
ipconfig测试

**Reply by Enigma-Soul** [reply]
好处是不用打洞

**Reply by Chiloven945** [reply]
我ping我的ipv6全部超时捏

**Reply by Enigma-Soul** [reply]
你路由器没开IP6
![屏幕截图 2023-04-08 205327](https://user-images.githubusercontent.com/119654171/230722260-8cda4c6c-f9f0-43d8-8cd2-9ed91dc8f1fe.png)

**Reply by Chiloven945** [reply]
一般人没有特殊需求不会去开ipv6因为他的用途较少，而且还要去设置很多人可能就卡这一步了，所以说还是老老实实用v4比较好（
~而且我连我路由器管理密码都忘了~

**Reply by Enigma-Soul** [reply]
但是IP4要打洞，

> 需要服务器

**Reply by Enigma-Soul** [reply]
这两年ip4公网地址快用完了，以后都是IP6，现在不用以后用

**Reply by Chiloven945** [reply]
我当然知道要用完了，但是现在大多数人仍然在使用ipv4而不是ipv6，并且在出现"需要ipv6来联机"这个事件之前他们的需求内并没有ipv6，他们的需求可能也只局限于能流畅上网，而且很多人说不定根本不知道这些是什么或者能带给他们什么，如果强行使用ipv6的话那么他们就无法使用联机的功能，或者是很麻烦去使用这个功能，所以说与其去使用麻烦用户的方法，不如采用对于用户方更加简便的方法

**Comment by Pigeon0v0** [comment]
联机在做的，不要急

**Reply by Enigma-Soul** [reply]
3个月前你们对我说的这句话

**Reply by Enigma-Soul** [reply]
帮帮龙猫吧

**Reply by WForst-Breeze** [reply]
您的急切之心我们能理解，但俗话说的好，心急吃不了热豆腐……
